rpc/internal/service_test.go

```diff
@@ -316,3 +325,101 @@ func TestAttachResultData(t *testing.T) {
 		})
 	}
 }
+
+func TestService(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	curatorPath, err := filepath.Abs("curator")
+	require.NoError(t, err)
+	for _, test := range []struct {
+		name    string
+		skip    bool
+		local   bool
+		setUp   func(t *testing.T)
+		cleanUp func(t *testing.T)
+		closer  func(t *testing.T)
```

Comment:
I think you can probabaly merge cleanup and closer, but it means that your functions need to user require.